### DISCUSSION TITLE
New API Method Not Found

### DISCUSSION THREAD

**Discussion by oksuzgil** [mynaparrot/plugNmeet-server#322] [General]
Hello, I have updated the system with update.sh and I am trying to add analytics to api queries, but the newly added methods are not coming to the api, it is not found.

Analytics

**Comment by jibon57** [comment]
This version hasn't published yet

**Reply by jibon57** [reply]
You can try demo server for testing. Feedback are welcome

**Reply by oksuzgil** [reply]
Hello Gibon, I am writing what I can see as feedback.

I ran tests for demo.plugnmeet.com.
"file_id": "RM_SzNwV4dtwQer-240-1693809377"

1- The meanings of the data in the analysis file should be written in the documentation. For example, is room_duration minutes or hours?

2- "ANALYTICS_STATUS_STARTED" value is returned in the mic_status tag, what is this?

3- In general terms, instead of downloading the json file for the relevant record, if it can be returned as json directly, the data can be printed on the screen by listing the log in the interface development environment, a separate method should be developed.

4- For interface design such as dashboard, data is required in the range of days. it seems difficult to read all the files in this way, arrange them according to the date range and download and interpret the files. Saving the files by dividing them by dates in the folder determined on the server side can provide convenience in scanning only the relevant day with a method to be written. (must return direct json)(/plugnmeet/logs/20230904/)

It was a very good idea, when you put it live, I can try it on our side and continue with the feedback.

**Reply by jibon57** [reply]
Another API to fetch meeting info: https://www.plugnmeet.org/docs/api/room/fetch-past
Contributors are welcome to describe files. You can check code to find which one for what purposes. At present don't have time to add details.

**Reply by oksuzgil** [reply]
I am aware that you do not have time and you are developing with limited people. Mine is to offer suggestions and feedback as always.
